tests/test_allow_zeros.py

```
@@ -0,0 +1,167 @@
+    return int(
+        datetime.strptime(date_str, "%Y-%m-%d")
+        .replace(tzinfo=timezone.utc)
+        .timestamp()
+    )
+
+
+def _safe_destroy(client: TNClient, stream_id: str) -> None:
+    try:
+        client.destroy_stream(stream_id)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)